ext/pdo_pgsql/pgsql_driver.c

```diff
@@ -102,9 +102,16 @@ int _pdo_pgsql_error(pdo_dbh_t *dbh, pdo_stmt_t *stmt, int errcode, const char *
 }
 /* }}} */
 
-static void _pdo_pgsql_notice(pdo_dbh_t *dbh, const char *message) /* {{{ */
+static void _pdo_pgsql_notice(void *context, const char *message) /* {{{ */
 {
-/*	pdo_pgsql_db_handle *H = (pdo_pgsql_db_handle *)dbh->driver_data; */
+	zval zarg;
```

Comment:
you do not have to do this change at all but little nit ; better to limit the scope of a var inside its useful block.